### PR TITLE
Fixed codacy problems

### PR DESCRIPTION
**Pull request by vgonisanz**
Added some corrections to JsonManager to avoid errors.

### CODE REVIEW

File: managers/jsonManager.py [hunk 1]
```diff
@@ -65,15 +68,17 @@ def test(self):
     """
     set_key_array
     """
-    def set_key_array(self, key_array=_key_array):
+    #@classmethod
+    def set_key_array(self, key_array = _key_array):
         print("Setting keys: %s" % list(key_array) )
         _key_array = key_array
         return
```
Comment (RDCH106) [inline managers/jsonManager.py]: What is the function of this return?

File: managers/jsonManager.py [hunk 2]
```diff
@@ -65,15 +68,17 @@ def test(self):
     """
     set_key_array
     """
-    def set_key_array(self, key_array=_key_array):
+    #@classmethod
+    def set_key_array(self, key_array = _key_array):
```
Comment (RDCH106) [inline managers/jsonManager.py]: It has no sense a default value for a setter.

Comment (RDCH106) [reply managers/jsonManager.py]: Please check the issue in Codacy:

https://www.codacy.com/app/RDCH106/SimplePythonTools/file/4324678323/issues/source?bid=3859861&fileBranchId=3860746#l72

**Comment by RDCH106** [comment]
Thanks for your contribution!

I have marked a pair of comments in your code of PR. 
There are code duplications, generate a function to reuse the code. It is the best way to improve the maintainability of the code.

**Comment by vgonisanz** [comment]
Branch ready to merge

**Review by RDCH106 — COMMENTED** [review]
Thank you for your corrections!

**Pull request merged**